tests/sentry/relay/config/test_metric_extraction.py

```diff
@@ -431,6 +431,105 @@ def test_get_metric_extraction_config_with_apdex(default_project):
         }
 
 
+@django_db_all
+@pytest.mark.parametrize("quality", ["good", "meh", "poor", "any"])
+@pytest.mark.parametrize("measurement", ["measurements.lcp"])
+def test_get_metric_extraction_config_with_count_web_vitals(default_project, quality, measurement):
+    with Feature({ON_DEMAND_METRICS: True, ON_DEMAND_METRICS_WIDGETS: True}):
+        create_alert(
```

Comment:
FYI this tests the support via creating alerts rather than widgets. Not sure if intentional.

Reply:
Wasn't intentional, I missed that they were different since the params are almost identical 🤔